independent-publisher-connectors/AADInvitationManager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,193 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure AD Invitation Manager",
+    "description": "A connector for the Azure AD Invitation Manager.",
```

Comment:
@Laskewitz Can you please provide a more detailed description ?

Docs - [Description of connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

Reply:
@sriyen-msft fixed!

Reply:
@Laskewitz Connector's description should describe concisely the main purpose and value offered by your connector. The give description is not enough to understand what is it created for. Please provide detailed explanation about `Azure AD Invitation Manager`.

Reply:
@sriyen-msft in a later commit I fixed the description for the connector. See: https://github.com/microsoft/PowerPlatformConnectors/pull/924/commits/2398a7be54af473c206258aa61dd02c66122fa3b